relayer/src/supervisor.rs

```diff
@@ -361,17 +356,6 @@ impl Supervisor {
                         IdentifiedConnectionEnd::new(connection_id.clone(), connection_end);
 
                     for channel in connection_channels {
-                        if !self.relay_on_channel(&chain_id, &channel.port_id, &channel.channel_id)
```

Comment:
Hmm removing these lines we would still spawn the uni-chan-path worker when Hermes starts. This worker would never receive any subsequent commands, but it can do packet clearing, for instance.

To see this in action, run with 
```
[chains.packet_filter]
policy = 'deny'
list = [
  ['transfer', 'channel-0'],
]
```
for ibc-0.

Then do `hermes start`, and we see


> Jul 07 16:10:44.875  INFO [ibc-0:transfer/channel-0 -> ibc-1] clearing old packets
Jul 07 16:10:44.887  INFO [ibc-0:transfer/channel-2 -> ibc-1] finished clearing pending packets
Jul 07 16:10:44.889  INFO [ibc-0:transfer/channel-3 -> ibc-1] finished clearing pending packets
Jul 07 16:10:44.889  INFO [ibc-0:transfer/channel-1 -> ibc-1] finished clearing pending packets
Jul 07 16:10:44.891  INFO [ibc-0:transfer/channel-4 -> ibc-1] finished clearing pending packets
Jul 07 16:10:44.898  INFO [ibc-0:transfer/channel-0 -> ibc-1] generate messages from batch with 1 events
Jul 07 16:10:44.915  INFO [ibc-0:transfer/channel-0 -> ibc-1] scheduling op. data with 1 msg(s) for Destination (height 0-13122)
Jul 07 16:10:44.926  INFO [ibc-0:transfer/channel-0 -> ibc-1] finished clearing pending packets
Jul 07 16:10:44.936  INFO [ibc-0:transfer/channel-0 -> ibc-1] relay op. data of 1 msgs(s) to Destination (height 0-13122), delayed by: 20.619838ms [try 1/5]
Jul 07 16:10:44.936  INFO [ibc-0:transfer/channel-0 -> ibc-1] prepending Destination client update @ height 0-13122
Jul 07 16:10:44.960  INFO [ibc-0:transfer/channel-0 -> ibc-1] assembled batch of 2 message(s)
Jul 07 16:10:46.754  INFO [ibc-0:transfer/channel-0 -> ibc-1] result events:
	UpdateClientEv(h: 1-13114, cs_h: 07-tendermint-0(0-13122))
	WriteAcknowledgementEv(WriteAcknowledgement - h:1-13114, seq:5, path:channel-0/transfer->channel-0/transfer, toh:1-13884, tos:Timestamp(NoTimestamp)))